packages/node-api/integration/Adapters.ts

```diff
@@ -21,6 +22,7 @@ export async function toSponsorshipId() {
 
 export async function toPalletLogionLocMetadataItem() {
     const { api } = await setup();
+    const polkadotAddress = encodeAddress("5FniDvPw22DMW1TLee9N8zBjzwKXaKB2DcvZZCQU5tjmv1kb", api.queries.ss58Prefix);
```

Comment:
What about moving this logic to `getValidAccountId`? The returned object could make address comparisons etc prefix-safe.